src/classes/js/parts/WrongFilePathLinter.ts

```diff
@@ -24,7 +24,10 @@ export class WrongFilePathLinter<
 				const manifest = ParserPool.getManifestForClass(UIClass.className);
 				if (manifest) {
 					const rClassNamesRegex = new RegExp(
-						`${manifest.componentName.replace(/\./, "\\.")}\\..*?(?="|'|}|\\[|\\]|>|\\|)`,
+						`${manifest.componentName.replace(
+							/\./g,
+							"\\."
+						)}\\.([a-zA-Z]|\\.)*?(?=("|'|\`|}|\\[|\\]|>|\\|))`,

```

Comment:
I think this could also support numbers (and maybe some special characters too?):

```suggestion
						)}\\.([a-zA-Z0-9_]|\\.)*?(?=("|'|\`|}|\\[|\\]|>|\\|))`,
```

Reply:
Agree regarding underscore and numbers. Not the best practices of naming I guess, but still possible. Regarding the special characters, not sure. Never saw such naming style anywhere, which special characters would make sense? Maybe `$` and that would be it.